src/Sylius/Behat/Context/Setup/ShopSecurityContext.php

```diff
@@ -67,4 +67,24 @@ public function iAmLoggedInCustomer(?string $fullName = null): void
 
         $this->sharedStorage->set('user', $user);
     }
+
+    /**
+     * @Given I am a logged in customer by using remember me option
+     */
+    public function iAmLoggedInCustomerByUsingRememberMeOption(?string $fullName = null): void
```

Comment:
```suggestion
    /**
     * @Given I am a logged in customer by using remember me option
     */
    public function iAmLoggedInCustomerByUsingRememberMeOption(): void
```
It is not used right now, there is no point in adding it in advance.